.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
Well done adding the "test.md" to the ".gitignore" 👏🏼 however the `test.md` is still appearing in the repo root directory as shown in the following screenshot
![image](https://user-images.githubusercontent.com/68030297/226976494-e4940811-bc5c-485b-89e6-fc343c984331.png)

- [ ] Please ensure that the `test.md` file is ignored and it should be missing from the remote repo and you can solve this issue by following the steps below:

1. First you need to clear the git tracking changes for this file by running the command `git rm --cashed path/your_file_name.extension`
2. Then add the changes by running `git add .`, then commit the changes.
3. don't forget to push the changes 👍🏼 

Hope this was helpful 💪🏼 good luck.